src/utils.ts

```diff
@@ -284,13 +285,15 @@ const queryStringParser = (
         urlParams = queryStringParserFallback(url);
     }
 
+    urlParams.forEach((value, key) => {
+        lowerCaseUrlParams[key.toLowerCase()] = value;
+    });
+
     if (isEmpty(keys)) {
-        urlParams.forEach((value, key) => {
-            results[key] = value;
-        });
+        return lowerCaseUrlParams;
     } else {
         keys.forEach(key => {
-            const value = urlParams.get(key);
+            const value = lowerCaseUrlParams[key];
```

Comment:
add .toLowerCase() to `key`, then revert the casing of sccid above and in the tests